demosplan/DemosPlanCoreBundle/Logic/Faq/FaqCategoryInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
Nearly the same points as mentioned for PlatformFaq.php.

Reply:
done in 9a35ce471ecbaa9ad940ebd8954ce70f55999a12